.changeset/fresh-crabs-deliver.md

```diff
@@ -0,0 +1,12 @@
+---
+"@spectrum-css/breadcrumb": major
+---
+
+#### Spectrum 2 Migration
```

Comment:
Should we mention the addition of the title sizing feature? for the multiline variant?

Reply:
Yes, good idea. Added.